### DISCUSSION TITLE
How To Add ID Metadata When Importing

### DISCUSSION THREAD

**Discussion by chaplaindan** [AntlerVC/firetable#1328] [Support / Q&A] [answered]
I am really enjoying the process of learning Rowy. **It's a great product and has a lot of depth.** 

I created a Rowy.io project and created a table called 'company' which had only two columns: 1. companyName and, 2. companyId. The companyId column is using a field type set to metadata ID. 

When I add a new row and type something in companyName, Rowy.io automatically adds an ID in in the companyId cell. 
But **when I import a list of companies in the table using CSV**, it doesn't add anything in the companyId cell. 

Q: when importing data into a table with an ID field, do you have to provide your own ID or is there a method to add/update the IDs afterward?

**Comment by BeeBombshell** [comment, marked as answer]
Hey @chaplaindan, the ID field type by default displays the Document ID in your Firestore Collection. If you wish to set your Document IDs manually, check out our docs on how you can set custom Document IDs: https://docs.rowy.io/how-to/add-row#modes-for-id-generation-while-adding-a-row.
When you're importing data, you can pick a column to be set as the Document ID for each row: https://docs.rowy.io/import-export-data/import-csv#autogenerated-id-vs-custom-id.

Hope this answers your question! :)
